src/one_dragon/base/push/channel/fake.py

```
@@ -0,0 +1,49 @@
+        self,
+        config: dict[str, str],
+        title: str,
+        content: str,
+        image: MatLike | None = None,
+        proxy_url: str | None = None,
+    ) -> tuple[bool, str]:
+        """
+        推送消息到钉钉机器人
+
+        Args:
+            config: 配置字典，包含 SECRET 和 TOKEN
```

Ruff (0.14.0)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)